### PR TITLE
Fixes hivemind weeding

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

I am going to stab splinter with a rusty nail for using a space

## Changelog
:cl:
fix: Hvemind can weed again
/:cl:

### CODE REVIEW

**Review by Hughgent — APPROVED** [review]

**Comment by ilikesaying** [comment]
is he going to get a _**splinter?**_

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 1]
```diff
@@ -93,13 +93,13 @@
...
+		return ..()
 	var/mob/living/carbon/xenomorph/hivemind/hiveminde = owner
-	if(!locate(/obj/effect/alien/weeds) in range(1, hiveminde.loc))
-		hiveminde.forceMove(get_turf(hiveminde.core))
-		to_chat(hiveminde, "<span class='xenonotice'>We can't place weeds with no weeds nearby, we got moved to our core.</span>")
-		return
- 	return ..()
- 
+	hiveminde.forceMove(get_turf(hiveminde.core))
+	to_chat(hiveminde, "<span class='xenonotice'>We can't place weeds with no weeds nearby, we got moved to our core.</span>")
```
Comment (Shadow-Quill) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:100]: why is it hiveminde?

Comment (Shadow-Quill) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:100]:
```suggestion
	to_chat(hiveminde, "<span class='xenonotice'>We can't place weeds with any weeds nearby, so we got moved to our core.</span>")
```

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/abilities.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:100]: But its with no weeds nearby

Comment (SplinterGP) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:100]: Hivemind cringe, hiveminde ok.

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 2]
```diff
@@ -93,13 +93,13 @@
 	cooldown_timer = 12 SECONDS
 
 /datum/action/xeno_action/plant_weeds/slow/action_activate()
+	if(locate(/obj/effect/alien/weeds) in range(2, owner.loc))
+		return ..()
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:97]:
```suggestion
	if(locate(/obj/effect/alien/weeds) in range(1, owner.loc))
		return ..()
```

**Comment by SplinterGP** [comment]
> is he going to get a _**splinter?**_

I'll need it 😿

File: code/modules/mob/living/carbon/xenomorph/abilities.dm (hunk 1 above)
Comment (Shadow-Quill) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:100]: Any would be correct here.

**Pull request merged**